Packs/FiltersAndTransformers/Scripts/RemoveMatches/README.md

```diff
@@ -0,0 +1,48 @@
+Removes items from the given list of values if they match any of the patterns in the provided `filters`.
+  If the match_exact argument is 'yes', direct string compare is used, otherwise the comparison is done using regex.
```

Comment:
```suggestion
Removes items from the given list of values if they match any of the patterns in the provided `filters`.
If the match_exact argument is 'yes', direct string compare is used, otherwise the comparison is done using regex.
```